release-notes/release-notes59.md

```diff
@@ -320,7 +320,7 @@ The new versions of the server-side services provide updates for the Java-based
         - 8.0.42+
         - 7.0.76+
 
-2. Replace the existing server-side `.war` file with the version 2.110.0 `.war` file bundled with {{site.productname}} 5.9 or later.
+2. Replace the existing server-side `.war` file with the version 2.110.? `.war` file bundled with {{site.productname}} 5.9 or later.
```

Comment:
- [ ] Don't merge it like this

None of the server-side components that are coming out with this release have a version number that looks like `2.110.0` that I can tell. Also, they all have different version numbers. I figured it was worth raising this as a "what are we doing" as part of this PR so that I can fix it if it needs fixing

Reply:
Yeah, they will all have different versions. Spelling will be 2.117.x and the others are 2.105.x (or something similar). The source of truth is whatever is in tinymce-enterprise artifacts file. There might always be more patch releases as well, depending on QA.

Reply:
```suggestion
2. Replace the existing server-side `.war` file with the `.war` file bundled with {{site.productname}} 5.9 or later.
```